docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -79,24 +79,25 @@ Alluxio UFS root mount point in the Alluxio docker image:
   ```console
   $ docker run -v /alluxio_ufs:/opt/alluxio/underFSStorage   ...
   ```
-  Of course, you can choose different path `/alluxio_ufs` you like to mount but please make sure it
-  is writable.
+Of course, you can choose different path `/alluxio_ufs` you like to mount but please make sure it
+is writable.
+> Note: Alluxio v2.1 docker image runs as user `alluxio` by default. It has UID 1000 and GID 1000.  
 
 ## Launch Alluxio Containers for Master and Worker
 
-According to Alluxio architecture, the Alluxio clients (local or remote) need to communicate with
-both Alluxio master and Workers. Therefore it is important to make sure the client can reach out
+The Alluxio clients (local or remote) need to communicate with
+both Alluxio master and workers. Therefore it is important to make sure the clients can reach
```

Comment:
nit: the clients -> client

Reply:
Changed to "make sure clients can reach"